homeassistant/components/onvif/camera.py

```diff
@@ -256,7 +253,15 @@ def __init__(self, hass, config):
         except exceptions.ONVIFError as err:
             _LOGGER.error("Couldn't setup camera '%s'. Error: %s",
                           self._name, err)
-            return
+
+    async def setup_ptz(self):
```

Comment:
This is still a coroutine. The name doesn't make it a coroutine. It's the definition `async def` that makes it a coroutine.